make-release.sh

```diff
@@ -223,6 +240,19 @@ setupGitconfig() {
   export GITHUB_TOKEN="${CHE_BOT_GITHUB_TOKEN}"
 }
 
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-v'|'--version') CHE_VERSION="$2"; shift 1;;
+    '-dv'|'--dwo-version') DWO_VERSION="$2"; shift 1;;
+    '-p'|'--phases') CHE_PARENT_VERSION="$2"; shift 1;;
```

Comment:
uh, oh, this looks very wrong

Reply:
also we need flags for `RELEASE_CHE_PARENT` (=`false` or `true`) and 
`VERSION_CHE_PARENT` (=`7.15.0`)

Reply:
Ah, digging further I realized we don't ... well, not currently. followup in https://github.com/eclipse/che/issues/19732